app/views/IncomeTaxSummaryView.scala.html

```diff
@@ -82,7 +85,7 @@
 
                 <h2 id="incomeFromEmploymentHeading" class="govuk-heading-m under-line">@Messages("tai.incomeTaxSummary.employment.section.heading")</h2>
 
-                @incomeTaxSummaryIncomeSourceDetail(viewModel.employments, "employment", Messages("tai.incomeTaxSummary.incomeAmount.prefix"))
+                @incomeTaxSummaryIncomeSourceDetail(viewModel.employments, "employment", Messages("tai.incomeTaxSummary.incomeAmount.prefix"), taxCodeViewModel: Map[String, TaxCodeViewModel])
```

Comment:
no need to cast `: Map[String, TaxCodeViewModel]`

Reply:
sure